configure

```diff
@@ -1414,7 +1464,8 @@ fi
 
 # check if getcwd accepts (NULL,0) as argument
 checking "if getcwd(NULL,0) returns malloc'ed string"
-cat >"${tempsrc}" <<END
+if [ -z "$YASH_GETCWD_AUTO_MALLOC" ]; then
```

Comment:
Can you add this variable to the list of "Influential environment variables" in the help message?